app/controllers/application_controller.rb

```diff
@@ -1,9 +1,15 @@
 class ApplicationController < ActionController::Base
   include Pundit
+  # after_action :verify_authorized
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  # after_action :verify_authorized
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name email password role])
```

Comment:
we don't need to receive `role` as a parameter from a client, because it always could be `SuperDuperAdmin`